src/components/Plot.tsx

```diff
@@ -195,8 +194,6 @@ export function Plot(props: PlotProps) {
             id={svgId}
             className={svgClassName}
           >
-            <MarkerDefs />
```

Comment:
@targos is `MarkerDefs` used just in annotation arrow?

Reply:
I think, yes.

Reply:
perfect